supplementary_style_guide/glossary_terms_conventions/general_conventions/s.adoc

```diff
@@ -590,6 +590,17 @@ Use all lower case, unless used in a title or at the beginning of a sentence.
 
 *See also*:
 
+[discrete]
+[[softirq]]
+==== image:images/yes.png[yes] softirq (noun)
+*Description*: A _software interrupt request (softirq)_ is a deferrable kernel routine that performs the required actions in response to an interrupt. For example, softirqs clear the receive ring buffer of a network adapter after the reception of one or more packets.
```

Comment:
suggested rewrite: "...interrupt; for example, softirqs clear the receive ring buffer after a network adapter receives a packet."

IBM SG also doesn't like "perform," but there aren't many better options here. maybe change thus: "...routine that acts in response to..."?

Reply:
@jseseCCS , I used your suggestion, but I avoided the semicolon.
@Srivaralakshmi